### DISCUSSION TITLE
What is the 4444 port in Manual Cluster Installation/Load Balancing?

### DISCUSSION THREAD

**Discussion by Mohit-3196** [ant-media/Ant-Media-Server#5318] [FAQs] [answered]
I'm in the process of creating an Ant Media Server Cluster manually by following the guide available at https://antmedia.io/docs/guides/clustering-and-scaling/cluster-installation/.

However, I have noticed that there have been some changes in the architecture, particularly when setting up the NGINX Load Balancer. In my previous setup, I used to access Origin Nodes on port 443 and Edge nodes on port 5443. However, in the updated architecture, it appears that the cluster is now accessible on port 4444.

I'm wondering how the distinction between Origin and Edge nodes will be made in this new architecture.

**Comment by mekya** [comment, marked as answer]
Hi @Mohit-3196,

Thank you for the question. Let me tell the story. 

- We've changed the NGINX load balancer algorithm to send the request to the instance having least connection because there is a problem in load distribution in some scenarios. 
- This breaks the logging in the web panel because session should be authenticated in only one instance. However, each request is forwarded to a new instance in least connection algorithm.
- We then create a new port 4444 to access the web panel by using ip hash algorithm. 

In short, just use 4444 for logging in to the web panel. 

For publishing and playing, you can use the 443 and 5443 ports in the same way. In addition, with the latest improvements no need to specify the port number.  Nginx configuration and SDKs integrity forwards the publish request to the origin groups and play requests to the edge groups. 

I hope it's an answer for you. Please let me know if you have further questions and I can help with anything.
